src/foam/box/ClusterSyncRequestMessage.js

```diff
@@ -0,0 +1,28 @@
+/**
+ * @license
+ * Copyright 2017 The FOAM Authors. All Rights Reserved.
```

Comment:
2019

Reply:
User short form:
/**
 * @license
 * Copyright 2019 The FOAM Authors. All Rights Reserved.
 * http://www.apache.org/licenses/LICENSE-2.0
 */